packages/grafana-data/src/datetime/moment_wrapper.ts

```diff
@@ -48,6 +48,7 @@ export interface DateTimeDuration {
   minutes: () => number;
   seconds: () => number;
   asSeconds: () => number;
+  humanize: () => string;
```

Comment:
It would be good if we could manage without extending the moment wrapper since it is basically deprecated.